bindings/wasm/CMakeLists.txt

```diff
@@ -30,7 +30,14 @@ file(MAKE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/examples/built)
 # ensure that interface files are copied over when modified
 add_custom_target(js_deps ALL
         DEPENDS manifoldjs
-               ${CMAKE_CURRENT_SOURCE_DIR}/manifold*.d.ts)
+               ${CMAKE_CURRENT_SOURCE_DIR}/manifold*.d.ts
+               ${CMAKE_SOURCE_DIR}/patch-emscripten.sh)
+
+if(MANIFOLD_PAR STREQUAL "TBB")
+add_custom_command(
+        TARGET js_deps POST_BUILD
+        COMMAND ${CMAKE_SOURCE_DIR}/patch-emscripten.sh ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Why is the patch applied both here and in the manifold.yml file?

Reply:
The patch applied in `manifold.yml` is for patching the `test/manifold_test.js`, while the patch here is to patch the binding files. I can remove the `package.json` so running `test/manifold_test.js` does not require any patch. I added that mainly for testing and forgot to remove it.